CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Data summary/RelatedData - Fix sorting by referenced field.
```

Comment:
This fix description is too generic in my opinion.
You should detail in which case the sorting on belongsTo relationships was broken.

Reply:
I would start the description with:
```suggestion
- Has Many Relationships - ...
```